mesa/visualization/templates/modular_template.html

```diff
@@ -139,12 +164,30 @@
 <script>
 	// Set up the GUI
 	var gui = new dat.GUI();
-	var fps_control = gui.add(control, "fps", 0, 20).step(1); // fps slider, 0 to 20 in 1 increments 
+	var fps_control = gui.add(control, "fps", 0, 50).step(1); // fps slider, 0 to 20 in 1 increments
```

Comment:
Is there a reason to increase this to 50? That seems on the high end, and at least some models can't keep up with that frame-rate, which causes buggy behavior right now.